ras_backstage/__init__.py

```diff
@@ -15,6 +18,34 @@
 app.url_map.strict_slashes = False
 
 logger_initial_config(service_name='ras-backstage', log_level=app.config['LOGGING_LEVEL'])
+logger = logging.getLogger(__name__)
+
+
+def get_public_key():
+    headers = {
+        'Accept': 'application/json',
+    }
+
+    public_key_url = f'{app.config["UAA_SERVICE_URL"]}{"/token_key"}'
+    response = requests.get(public_key_url, headers=headers)
+
+    try:
+        response.raise_for_status()
+    except HTTPError:
+        logger.exception("Error while retrieving public key")
+        raise
+
+    try:
+        key = response.json()['value']
```

Comment:
Response.json raises a valueError if it fails.  Should be caught here on the very slim chance that this might happen
http://docs.python-requests.org/en/latest/user/quickstart/#json-response-content

Reply:
Though any error in __init__.py will stop the application dead